src/Http/Resources/Post/ListPostResourceCollection.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace CSlant\Blog\Api\Http\Resources\Post;
+
+use Illuminate\Http\Request;
+use Illuminate\Http\Resources\Json\ResourceCollection;
+use Illuminate\Pagination\LengthAwarePaginator;
+
+class ListPostResourceCollection extends ResourceCollection
```

Comment:
@pxthinh @TonyDuong0509 please research what the resource collection is and I need to use it another API.
thanks!